api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -316,6 +316,26 @@ message Pass {
   uint64 uplink_center_frequency_hz = 10;
 }
 
+// Data uploaded from the ground station during a pass.
+message TelemetryData {
+  // URL of the data. This URL is valid for one hour after being returned.
+  string url = 1;
+
+  // The type of data available at the above URL.
+  enum DataType {
```

Comment:
I believe this is the same as `FramingType` - RAW == IQ, DEMODULATED == BITSTREAM, DECODED == AX.25, etc.

Reply:
@hoshir can you comment on this?

I think I understand that RAW == IQ and DEMODULATED == BITSTREAM but I thought DECODED was more like getting an image, text file or whatever.

Reply:
After talking to Andrew I think I get it now. Decoded means that we've removed the framing data (e.g. AX.25 headers etc).

I think it would be confusing to mention framing type here.

Reply:
The enum values we want to use match with Framing enum, BITSTREAM, AX25, IQ (and IMG_PNG). However, using the enum to a variable named data_type sounds confusing to me. For me, changing variable name to something like "Framing data_framing" sounds natural when we reuse Framing enum.

Reply:
I noticed the framing enum also includes IMAGE_PNG and is used in requests too.

If we can freely add additional values for other decoded data types then it is convenient to reuse it. The name 'framing' feels not very accurate though.

I'll consider updating the API later.